db/migrate/20150916162754_add_log_path_to_services_table.rb

```diff
@@ -0,0 +1,5 @@
+class AddLogPathToServicesTable < ActiveRecord::Migration
+  def change
+    add_column :services, :log_path, :text
+  end
```

Comment:
@rahij Sounds reasonable, in that case the add new services page wouldn't need to set `log_path` anymore. I think that simplifies the whole app flow.